.github/dependabot.yml

```diff
@@ -18,3 +18,338 @@ updates:
         update-types:
           - "version-update:semver-major"
           - "version-update:semver-minor"
+
+  - package-ecosystem: nuget
```

Comment:
not sure if this or manual one is better...

Reply:
Either manual or monthly could be a better cadence here. Daily would increase maintenance burden.